ask-watch.py

```diff
@@ -3,8 +3,35 @@
 import ask
 import os
 
+
 os.system('clear')
-print('⚡' + '\033[92m' + '️Ask' + '\033[0m' + ' Watcher')
+
+
+def style_string(text,color='', bold=False):
+
+	ret = text
+	if color == 'red':
+		ret = '\033[91m' + ret + '\033[0m'
```

Comment:
You could create two variables: `prefix = '\033['` & `suffix = prefix + '0m'` at the top of the method to reduce duplicate code a bit.